x-pack/plugins/alerting/common/routes/r_rule/response/schemas/v1.ts

```diff
@@ -8,40 +8,157 @@
 import { schema } from '@kbn/config-schema';
 
 export const rRuleResponseSchema = schema.object({
-  dtstart: schema.string(),
-  tzid: schema.string(),
+  dtstart: schema.string({
+    meta: {
+      description: 'Rule start date as per UTC.',
+    },
+  }),
+  tzid: schema.string({
+    meta: {
+      description: 'Indicates timezone.',
+    },
+  }),
   freq: schema.maybe(
-    schema.oneOf([
-      schema.literal(0),
-      schema.literal(1),
-      schema.literal(2),
-      schema.literal(3),
-      schema.literal(4),
-      schema.literal(5),
-      schema.literal(6),
-    ])
-  ),
-  until: schema.maybe(schema.string()),
-  count: schema.maybe(schema.number()),
-  interval: schema.maybe(schema.number()),
+    schema.oneOf(
+      [
+        schema.literal(0),
+        schema.literal(1),
+        schema.literal(2),
+        schema.literal(3),
+        schema.literal(4),
+        schema.literal(5),
+        schema.literal(6),
+      ],
+      {
+        meta: {
+          description:
+            'Indicates frequency of the rule. Options are YEARLY, MONTHLY, WEEKLY, DAILY.',
+        },
+      }
+    )
+  ),
+  until: schema.maybe(
+    schema.string({
+      meta: {
+        description: 'Recur the rule until this date.',
+      },
+    })
+  ),
+  count: schema.maybe(
+    schema.number({
+      meta: {
+        description: 'Number of times the rule should recur until it stops.',
+      },
+    })
+  ),
+  interval: schema.maybe(
+    schema.number({
+      meta: {
+        description:
+          'Indicates the interval of frequency, e.g. 1 and YEARLY is every 1 year, 2 and WEEKLY is every 2 weeks.',
```

Comment:
We generally try to avoid using Latin abbreviations, so I've suggested "for example":
```suggestion
          'Indicates the interval of frequency. For example, 1 and YEARLY is every 1 year, 2 and WEEKLY is every 2 weeks.',
```